modules/mongodb-bi-connector/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "helm_release" "mongodb_bi_connector" {
```

Comment:
* please have versions.tf with terraform, helm providers versions constraints
* please have variable to set namespace, with default "default" namespace